### PR TITLE
Fixed card padding issue

### LINKED ISSUE #1443: Aesthetic Issue: Link boxes missing padding around text
I noticed recently that the feature boxes that are used to link to internal pages no longer have padding around the text.

Example: the boxes at the bottom of this page: https://jakarta.ee/about/working-group/

And in the sidebar of any of the specification pages: https://jakarta.ee/specifications/platform/

It should look more like this: https://invis.io/AX10M9QL74RN#/448585126_Specifications-_Detail_Page

Screenshots from Chrome on Windows.

Screenshot 2022-03-23 135734

### PR DESCRIPTION
**Pull request by linkfang**
Signed-off-by: Zhou Fang <zhou.fang@eclipse-foundation.org>

Fixes https://github.com/jakartaee/jakarta.ee/issues/1443

### CODE REVIEW

**Comment by linkfang** [comment]
Got +1 from Kristy, https://github.com/jakartaee/jakarta.ee/issues/1443#issuecomment-1080026945

**Comment by ericpoirier** [comment]
@linkfang with this PR, you're also affecting the blocks on this page:

https://deploy-preview-1449--jakartaee.netlify.app/specifications/

I think you should either use margin-top:30px; or create a new class for the titles on blue background so that it doesn't affect blocks on other pages.

If you remove the margin-bottom for blocks with white background, it looks a bit off.

**Comment by linkfang** [comment]
Updated the style based on Eric's feedback

**Review by ericpoirier — APPROVED** [review]

**Pull request merged**
